en/components/themes/sass/presets/material.md

```diff
@@ -42,21 +50,22 @@ We create those mixins just for your convenience, otherwise, if you want to use
 
 $primary-color: #2ab759;
 $secondary-color: #f96a88;
+$surface-color: #303030;
 
+// IMPORTANT, the primary, secondary and surface colors are required
+// The $gray will ensure that all the black text, borders, etc will become a shade of white in order to be visible against a dark background
 $my-color-palette: palette(
-    // IMPORTANT, the primary and secondary colors are required
-    $primary: $primary-color,
-    $secondary: $secondary-color,
-    // The $gray will ensure that all the black text, borders, etc will 
-    // become a shade of white in order to be visible against a dark background
-    $gray: #fff 
-    // Since all individual components use the $default-palette by default, if you don't specify the rest of the colors like $info and $error they will use their default values from the $default-palette
+  $primary: $primary-color,
+  $secondary: $secondary-color,
+  $surface: $surface-color,
+  $gray: #fff,
 );
 
 @include theme($my-color-palette, $schema: $dark-schema);
 ```
 
+Since all individual components use the `$default-palette` by default, if you don't specify the rest of the colors like $info and $error they will use their default values from the `$default-palette`.
```

Comment:
```suggestion
Since all individual components use the `$light-material-palette` by default, if you don't specify the rest of the colors like $info and $error they will use their default values from the `$light-material-palette`.
```